From d617675dc6f9f960fd37784ac4ba9b851498c182 Mon Sep 17 00:00:00 2001
From: LP <lovelesh_p@zohomail.in>
Date: Mon, 29 May 2023 13:19:26 +0530
Subject: [PATCH] feature1 added

---
 app.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/app.py b/app.py
index 222ff82..8498076 100644
--- a/app.py
+++ b/app.py
@@ -1,11 +1,11 @@
 import platform
 import psutil
 
-
 def get_system_info():
-    """Gets detailed system information."""
 
-    # Get the system name.
+    """ Gets detailed system information """
+
+    # Get the system name
     system_name = platform.system()
 
     # Get the node name.
@@ -35,7 +35,7 @@ def get_system_info():
     # Get the operating system architecture.
     architecture = platform.architecture()[0]
 
-    # Get the amount of free memory.
+    # Get the amount of free memory
     free_memory = round(psutil.virtual_memory().free / 1024 ** 3, 2)
 
     # Get the amount of used memory.
-- 
2.40.1.windows.1

